addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -8,15 +8,42 @@
     justify-content: space-between;
 }
 
+.o_ChatterTopbar_buttonAttachmentsCountLoader {
+    margin-left: 2px;
+}
+
 .o_ChatterTopbar_buttonCount {
-    padding-left: 0.25rem !important;
+    padding-left: 0.25rem;
 }
 
+.o_ChatterTopbar_buttonSendMessage {
+    padding: 0 16px;
+    border-left: none;
+
+    &.o-buttonActive {
+        border-left: none;
```

Comment:
overwrite style-rule with same style-rule? Specificity issue maybe?

Reply:
I think I remember it's due to bootstrap whose rule becomes more specific on active buttons and overrule the border-left ; I'll check that

Reply:
If that's the case, I prefer explicit higher specificity of style, like here:
https://github.com/odoo/odoo/blob/master/addons/im_livechat/static/src/scss/im_livechat_form.scss#L14

Reply:
Could not reproduce the problem with bootstrap so I think I'm confusing it with another situation. I removed this useless statement.